source/github-helpers/heat-map.css

```
@@ -4,42 +4,46 @@ Original file: https://github.githubassets.com/assets/code-d21b20a03009.css
 These colors are for normally-gray text.
 */
 
+:root {
+	--rgh-heat-color-deg: 29deg;
+}
+
 :root [data-rgh-heat='1'] {
-	color: #c24e00 !important;
+	color: hsl(var(--rgh-heat-color-deg) 100% 38%) !important;
```

It seems that Safari _does_ support `color-mix` and Firefox will soon too (it's been behind a flag for 20 versions)

So I'm thinking:

1. we ship `color-mix` _today_
2. we keep the hex values as fallbacks

```suggestion
	color: #c24e00 !important;
	color: color-mix(in srgb, var(--rgh-heat-color), --rgh-gray)) !important;
```

---

What is `--rgh-gray`?

---

@fregante done